django/thunderstore/community/models/community.py

```diff
@@ -19,10 +19,37 @@ def listed(self) -> "QuerySet[Community]":  # TODO: Generic type
         return self.exclude(is_listed=False)
 
 
+def get_community_filepath(instance: "Community", filename: str) -> str:
+    return f"community/{instance.identifier}/{filename}"
+
+
 class Community(TimestampMixin, models.Model):
     objects: "CommunityManager[Community]" = CommunityManager()
     members: "Manager[CommunityMembership]"
 
+    slogan = models.CharField(max_length=512, blank=True, null=True)
```

Comment:
[Django's documentation](https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.Field.null) recommends against using `null=True` on `CharField`s. As this is a copy-paste from another model, we might want to address the issue in a separate migration/commit.

Reply:
The docs make a good point but it doesn't mean it should be applied everywhere IMO. Quite often I've found it useful to be able to discern between an unset field vs. a defined but empty field, probably most commonly in scenarios where some form of inheritance is going to be applied (allow overriding of inherited value with an empty string instead of assuming empty to mean there's no override).

Not sure whether or not it'll end up being useful for this field in practice, but given that it's impossible to migrate backwards from a non-nullable field to a nullable field without losing information, I've habitually defaulted to allowing null values.